server/liveblog/themes/themes.py

```diff
@@ -600,24 +601,26 @@ def _get_publishing_step(self, theme):
                 return STEPS.get(key)
         return STEPS.get("default")
 
-    def _schedule_items(self, items, step, is_blog):
-        """Schedules publishing for given items."""
+    def _schedule_items(self, items, step, is_blog, batch_size=10, delay=1):
+        """Schedules publishing for given items in manageable batches."""
         from liveblog.blogs.tasks import publish_blog_embed_on_s3
 
         countdown = 1
+        total_batches = ceil(items.count() / batch_size)
```

Comment:
what is the type of items, is it a cursor? if so, don't you get a deprecation warning?

Reply:
No, I'm not getting a deprecation warning on this

Reply:
I've refactored this to use list instead.